### PR TITLE
 updating readme

### PR DESCRIPTION
**Pull request by L0wry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,38 +5,67 @@
 
 ## Purpose
 
-Dextrose cli aims to provide a simple way to get visual snapshots of specified components across all react native platforms.
+Dextrose cli integrates into developer workflow by  providing visual snapshots of react native components described by storybook across all react native platforms.
```
Comment (asifhafeez) [inline README.md]: extra space between 'by' and 'providing'

Comment (asifhafeez) [reply README.md]: Could you maybe explain what Storybook is here? Or would we be assuming people using dextrose would know? Maybe we could at least add a link to the Storybook github repo? (https://github.com/storybooks/storybook)

File: README.md [hunk 2]
```diff
@@ -5,38 +5,67 @@
...
-Dextrose extends [fructose] by creating a new client. This client has the ability to request the loaded components of a react native app wrapped with Fructose. Once it has the components it cycles through each component and creates an image dump.
+Dextrose extends the [fructose] react native wrapper app by creating a new client.
+It's purpose is to iterate through all the bundled components loaded in the app and take a screenshot of each.
 
-What you do with those images is up to you!
+Dextrose generates a temporary stories file which reuses your existing components storybook stories. This results in maintainable and seamless integration to your existing workflow.
 
+## Features
+- Images of components across native platforms
+- Images of components at specified breakpoints web
```
Comment (asifhafeez) [inline README.md]: do you mean 'for web only'?

File: README.md [hunk 3]
```diff
@@ -5,38 +5,67 @@
...
+- Upload to s3 Bucket
```
Comment (asifhafeez) [inline README.md]: Maybe make this a bit clearer. Does it automatically upload to an s3 bucket?

File: README.md [hunk 4]
```diff
@@ -5,38 +5,67 @@
...
+    clean-stories
 
+The Dextrose generate-stories command should be run before the run command in following way:
+   
+    dextrose generate-stories path/to/your/components/directory
 
 The Dextrose run command can be run with the following commands:
 
-    dextrose run --config path-to-config
+    dextrose run --config, -c -path-to-config
```
Comment (asifhafeez) [inline README.md]: Shouldn't this be '/path/to/config' instead of '-path-to-config'?

File: README.md [hunk 5]
```diff
@@ -5,38 +5,67 @@
...
+Web
+
+        module.exports = {
+            snapPath: path.join(__dirname, '../snaps'),
+            platformName: "web",
+            breakpoints:[500, 1000],
+            ignoredStories: ["IGNORE"]
+        }
+
 The Dextrose upload command can be run with the following commands:
```
Comment (asifhafeez) [inline README.md]: maybe mention that this uploads to an s3 bucket?

File: README.md [hunk 6]
```diff
@@ -65,39 +94,23 @@ The Dextrose generate-html command can be run with the following commands:
...
-        }
 
 ## Running the example
 To run the the example Pull the repo
 - Run `yarn` in the Dextrose folder
 - Run `yarn` in the snapshots folder (the snapshots folder is an example of how you would consume dextrose)
 - Run `./run-(ios | android | web).sh`
 
-note: please make sure you have an (emulator | simulator), (appium | selenium) and a packager running.
+note: please make sure you have an (emulator | simulator), and a packager running.
```
Comment (asifhafeez) [inline README.md]: Not sure what '(emulator | simulator)' means here

Comment (kaliabadi) [reply README.md]: if you are running android they have an emulator but for iOS you use simulators

Comment (asifhafeez) [reply README.md]: cool, makes sense

**Pull request merged**